vizro-core/src/vizro/models/_components/form/dropdown.py

```diff
@@ -110,19 +128,23 @@ def __call__(self, options):
         option_height = _calculate_option_height(full_options)
         altered_options = _add_select_all_option(full_options=full_options) if self.multi else full_options
 
+        defaults = {
+            "id": self.id,
+            "options": altered_options,
+            "value": self.value if self.value is not None else default_value,
+            "multi": self.multi,
+            "optionHeight": option_height,
+            "persistence": True,
+            "persistence_type": "session",
+            "className": "dropdown",
+        }
+
+        dropdown = dcc.Dropdown(**(defaults | self.extra))
+
         return html.Div(
             children=[
                 dbc.Label(self.title, html_for=self.id) if self.title else None,
-                dcc.Dropdown(
-                    id=self.id,
-                    options=altered_options,
-                    value=self.value if self.value is not None else default_value,
-                    multi=self.multi,
-                    optionHeight=option_height,
-                    persistence=True,
-                    persistence_type="session",
-                    className="dropdown",
-                ),
+                dropdown,
```

Comment:
```suggestion
                dcc.Dropdown(**(defaults | self.extra)),
```

Just for consistency with other files.